src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="App">
+      <Display />
+      <ButtonPanel />
+    </div>
+  );
```

Comment:
- [ ] It's better to wrap these two components using React.Fragment short syntax. Since there is no need to add an extra div node.